### PR TITLE
Update main.yml

### PR DESCRIPTION
**Pull request by erikskaar**
Testing ci update

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -9,11 +9,8 @@ jobs:
       run: | 
         cd frontend
         yarn add eslint -g 
-        yarn add eslint-plugin-react eslint-config-airbnb-typescript  --save-dev
         npx eslint --ext .ts,.tsx src/
         cd ../backend
-        yarn
```
Comment (AndreasWintherMoen) [inline .github/workflows/main.yml:15]: I think we have to keep this?

**Review by AndreasWintherMoen — CHANGES_REQUESTED** [review]
CI still fails. I think we have to either run `yarn` and install all packages or install the specific linting packages we need. I think installing all is the safest option.

File: .github/workflows/main.yml (hunk 1 above)
Comment (erikskaar) [reply .github/workflows/main.yml:15]: image

What do you mean? It says all checks have passed 🤔 tested with and without yarn and it didn't make a difference (as can be seen in the commits)

Comment (AndreasWintherMoen) [reply .github/workflows/main.yml:15]: Oh, looked at the wrong commit I guess. Alright

**Review by AndreasWintherMoen — APPROVED** [review]

**Pull request merged**